lib/diplomat/key_utils.ex

```diff
@@ -0,0 +1,200 @@
+defmodule KeyUtils do
```

Comment:
If I understand the python library properly, it seems the urlsafe stuff is just a convenience for initializing a Key from a base64-encoded reference. Is that something that we actually need in an Elixir implementation?

Can you explain the use case for this a bit more? When would you actually need to use this?